docker-compose/local/setup.sh

```diff
@@ -287,18 +287,30 @@ generate_key() {
 
 echo $(show_message "security_secrect_regenerate")
 
+# check operating system
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    # macOS
+    SED_COMMAND="sed -i ''"
+elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
```

Comment:
Is it possible to only modify the `sed` command for macos and don't change for other system?

Reply:
sure
will change to this:
`if [[ "$OSTYPE" == "darwin"* ]]; then
    # macOS
    SED_COMMAND="sed -i ''"
else
    SED_COMMAND="sed -i"`
    
change nothing in other systems